tests/components/mqtt/test_fan.py

```diff
@@ -435,15 +435,6 @@ async def test_sending_mqtt_commands_and_explicit_optimistic(hass, mqtt_mock):
     assert state.state is STATE_OFF
     assert state.attributes.get(ATTR_ASSUMED_STATE)
 
-    await common.async_set_speed(hass, "fan.test", "cUsToM")
```

Comment:
We should test that an invalid speed in the service call raises ValueError with `pytest.raises`.

Reply:
Nice. I pushed the change right before I read your comment :)